src/client/main.tsx

```diff
@@ -4,9 +4,15 @@ import { GameCanvas } from './game-canvas';
 import { GameState } from '../core/game';
 import { ClientController } from './client-controller';
 import { Splash } from './splash';
+import { Lobby } from './lobby'
+import { GAME_START_TIME, SEND_NEW_PLAYER_JOINED } from '../server/server-interface'
 
+const GAME_START_TIME_IN_SECONDS = GAME_START_TIME / 1000;
```

Comment:
lets just do `GAME_START_TIME / 1000` where its called, no point having an extra variable thats even more characters than what it represents